hibernate-core/src/main/java/org/hibernate/tool/schema/internal/exec/ScriptTargetOutputToUrl.java

```diff
@@ -36,10 +37,23 @@
 	 *
 	 * @param url The url to read from
 	 * @param charsetName The charset name
+	 * @param append If true, then bytes will be written to the end of the file rather than the beginning
 	 */
-	public ScriptTargetOutputToUrl(URL url, String charsetName) {
+	public ScriptTargetOutputToUrl(URL url, String charsetName, boolean append ) {
```

Comment:
space delimiter after 'append' unnecessary and inconsistent here.

Reply:
Thanks @NathanQingyangXu , fixed it!